.github/ISSUE_TEMPLATE/documentation_request.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Documentation request
+about: Missing documentation or suggest an improvement
+
+---
+
+<!--
+Thanks for submitting this documentation request. Please fill the template
+below, otherwise we will not be able to process this request.
+
+Open an issue with this template if you think documentation regarding a certain
+aspect(s) is lacking or missing, or if you have an idea on how
+to improve the documentation.
+-->
+
+#### Summary
+<!-- Summarize the suggestion in a few sentences: -->
+
+...
+
+#### Why do we need this ?
+<!-- Please explain the motivation, for whom, etc. -->
+
+...
+
+#### What is already there? What do you see now?
+<!--
+Please add any relevant documentation article and resources. Screenshots if necessary.
+"Nothing available" is a valid answer.
```

Comment:
We don't have this line in our feature request either, so let's also not put it in here.